pkg/config/config.go

```diff
@@ -26,7 +26,7 @@ type Config struct {
 	GoProxy          string    `envconfig:"GOPROXY"`
 	GoBinaryEnvVars  EnvList   `envconfig:"ATHENS_GO_BINARY_ENV_VARS"`
 	GoGetWorkers     int       `validate:"required" envconfig:"ATHENS_GOGET_WORKERS"`
-	GoGetDir         string    `envconfig:"ATHENS_GOGOET_DIR"`
+	GoGetDir         string    `envconfig:"ATHENS_GOGET_DIR"`
```

Comment:
This is unfortunately a breaking change. But I think it's a necessary one. Thanks for the catch!

Reply:
Yes, I was thinking it might affect the users of Athens who have configured their environment variables to use the older name in their scripts/automation.

Reply:
You're most welcome! Athens is a great project and we@AWS have had a great experience using it. Thank you!